fern/versions/v1/pages/api-backend/sdk-1.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Software Development Kits"
+slug: "sdk"
+subtitle: ""
+hidden: false
+createdAt: "Wed Jun 01 2022 23:44:11 GMT+0000 (Coordinated Universal Time)"
+updatedAt: "Tue Dec 12 2023 22:09:32 GMT+0000 (Coordinated Universal Time)"
```

Comment:
These are not correct, does that matter?

Reply:
i don't think so..